drivers/mpu9250/MPU9250.hpp

```diff
@@ -178,8 +178,13 @@ namespace DriverFramework
 #define M_PI_F 3.14159265358979323846f
 #endif
 
+#if defined(__EDISON)
+// update frequency 100 Hz
+#define MPU9250_MEASURE_INTERVAL_US 10000
```

Comment:
I would suggest to update at 250 Hz. The estimator and control loops on Pixhawk and Snapdragon platforms run at 250 Hz and the performance at 100 Hz will suffer.